p4sdk/images/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Intel Corporation
+FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as builder
```

Comment:
update Hadolint ci job to ignore for this DL3029 warning as the --platform is required

Reply:
this is now resolved in main